gpt-survey-summarizer/README.md

```diff
@@ -34,7 +67,7 @@ A helm chart for the gptSuverySummarizer
 | config.discord.guildId | string | `""` | Discord Guild/Server ID |
 | config.discord.token | string | `""` | Discord API Token |
 | config.openAiApiKey | string | `""` | Openai API Key |
-| config.summarizeFrequencySeconds | int | `3600` | Summarize Frequency Seconds |
+| config.summarizeFrequencySeconds | int | `3600` | Summarize Frequency Seconds test |
```

Comment:
**suggestion:** We should re-run the pre-commit hook here, to get the README based on the template.